header/script.js

```diff
@@ -0,0 +1,65 @@
+window.Header = {
+  bind : function(){
+    var headers = document.querySelectorAll('.header');
+    var self = this;
+    for (var i = 0; i < headers.length; i++) {
+    if(headers[i].config !== undefined) continue;
+      var config = {};
+      config.scrollHeight = Number(headers[i].getAttribute('scrollHeight'));
+      config.topClasses = headers[i].getAttribute('topClasses');
+      config.bottomClasses = headers[i].getAttribute('bottomClasses');
+      config.enableTransition = headers[i].getAttribute('enableTransition');
+      self.create(headers[i], config);
+    }
+  },
+  create: function(elm, config){
+    if(!elm.classList.contains('header')){
+      elm.classList.add('header')
+    }
+    config.scrollHeight = config.scrollHeight || 100;
+    config.topClasses = config.topClasses || "";
+    config.topClasses = config.topClasses.split(" ");
+    config.bottomClasses = config.bottomClasses || "";
+    config.bottomClasses = config.bottomClasses.split(" ");
+    config.enableTransition = config.enableTransition === undefined || config.enableTransition === null || config.enableTransition === "true" || config.enableTransition === "1" || config.enableTransition === true;
+    if(elm.config) {
+      for(x in elm.config.bottomClasses)
+        elm.classList.remove(elm.config.bottomClasses[x]);
+    for(x in elm.config.topClasses)
+        elm.classList.remove(elm.config.bottomClasses[x]);
+    }
+    elm.config = config;
+    elm.parentNode.onscroll = () => {
```

Comment:
For use out content you need this code:
```suggestion
elm.parentNode.querySelectorAll('.content')[0].onscroll = function () {
```
And not use Arrow Function because same WebView Android not support this feature